Update Dockerfile for JDK 17.

R-Jellema opened the pull request:
Line 3: Toegevoegd zodat docker image updated is voor updates voor debian en debian download repository lists, anders geen JDK 17 mogelijk.

Line 27/28 - 31/32: Veranderd JDK15 -> JDK17 om zo de nieuwe JDK17 op dockerimage te krijgen.

R-Jellema committed:
Update Dockerfile for JDK 17.

Line 3: apt-get... toegevoegd zodat docker image updated is voor nieuwste debian versie, anders geen JDK 17 mogelijk.

Line 27/28 - 31/21: Veranderd JDK15 -> JDK17 om zo de nieuwe JDK17 op dockerimage te krijgen.

rodymiddelkoop reviewed (changes requested):
Ik wil graag dat JDK15 gehandhaafd blijft en JDK8 ook. JDK17 mag _ernaast_ gezet worden, niet in plaats van JDK15.

Preconditie: JDK8 en JDK15 aanwezig
Postconditie: JDK8 en JDK15 werken nog steeds identiek en JDK17 is toegevoegd

R-Jellema commented:
> Ik wil graag dat JDK15 gehandhaafd blijft en JDK8 ook. JDK17 mag _ernaast_ gezet worden, niet in plaats van JDK15.
> 
> Preconditie: JDK8 en JDK15 aanwezig Postconditie: JDK8 en JDK15 werken nog steeds identiek en JDK17 is toegevoegd

JDK 8 stond er niet in. JDK 15 wel. Ik kan JDK 15 terug zetten maar weet u dan zeker dat JDK 8 er ook in hoort?

R-Jellema commented:
![afbeelding](https://user-images.githubusercontent.com/19806733/142213949-1969072e-25fb-4e90-a88d-1470e37a09fc.png)

JDK 15 is niet meer beschikbaar vanaf deze download repo, daarom ook de switch naar JDK17. Ik wil wel een andere repo toevoegen voor 8/15 (als ik die kan vinden) maar dan wel met toestemming van u @rodymiddelkoop. Of OpenJDK als dat ook mag.

R-Jellema committed:
Update dockerfile | OpenJDK 15, JDK 12 en 17.

OpenJDK 15 toegevoegd. JDK 12 zit er nog bij. Kan dit niet terug vinden als installatie in de DockerFile maar staat er wel bij.
JDK 17 installatie in DockerFile

R-Jellema commented:
![Screenshot from 2021-11-17 17-10-09](https://user-images.githubusercontent.com/19806733/142237924-92e0601b-ba53-4c0a-a165-09f247247b41.png)

JDK 12 is al standaard geinstalleerd, kan de installatie commando's niet vinden in de DockerFile hiervan.
OpenJDK 15 installatie commando's.
